crates/red_knot_python_semantic/resources/mdtest/declaration/async_with.md

```diff
@@ -0,0 +1,20 @@
+# Async with declarations
+
+## Basic `async with` statement
+
+The type of the target variable in a `with` statement should be the return type from the context manager's `__aenter__` method.
+However, Knot doesn't support `async with` statements yet. This test asserts that it doesn't emit any context manager-related errors.
```

Comment:
nit: quite long lines here. Also, we haven't renamed it to `Knot` _yet_ ;)

```suggestion
The type of the target variable in a `with` statement should be the return type
from the context manager's `__aenter__` method. However, we don't support `async with`
statements yet. This test asserts that it doesn't emit any context manager-related errors.
```

Reply:
I'm open to enabling markdown formatting but I otherwise prefer not to reflow text manually. This does not affect how the text is rendered

Reply:
> This does not affect how the text is rendered

I know. My concern is more the ability to read and edit the code on small screens. But I agree it's not the most important issue in the world, just a nit.

Reply:
https://github.com/astral-sh/ruff/pull/14020